### ISSUE TITLE
Getting 404 for first-time publish attempt

### ISSUE THREAD

**Issue by IdanAdar** [JS-DevTools/npm-publish#23]
My package doesn’t exist yet. I want to use this action to publish it... but I get 404.

How do I perform an initial release?
@JamesMessinger 

I am also curious about this. Why would it use “latest”? Shouldn’t it use “0.0.1” per the package.json?

npm ERR! 404 '@ibm-cloud/secrets-manager@latest' is not in the npm registry.

Complete log: 
```
Error: Unable to determine the current version of @ibm-cloud/secrets-manager on NPM. 
npm view @ibm-cloud/secrets-manager version exited with a status of 1.

npm ERR! code E404
npm ERR! 404 '@ibm-cloud/secrets-manager' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 404 
npm ERR! 404  '@ibm-cloud/secrets-manager@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-11-24T07_42_31_223Z-debug.log
    at Object.getLatestVersion (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/src/npm.ts:57:13)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

ProcessError: npm view @ibm-cloud/secrets-manager version exited with a status of 1.

npm ERR! code E404
npm ERR! 404 '@ibm-cloud/secrets-manager' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 404 
npm ERR! 404  '@ibm-cloud/secrets-manager@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-11-24T07_42_31_223Z-debug.log
    at normalizeResult (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/node_modules/@jsdevtools/ez-spawn/lib/normalize-result.js:31:1)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/node_modules/@jsdevtools/ez-spawn/lib/async.js:79:1)
    at ChildProcess.emit (events.js:210:5)
```

**Comment by IdanAdar** [comment]
Maybe related to this? https://github.com/JS-DevTools/npm-publish/pull/19

**Comment by JamesMessinger** [comment]
#19 has been merged and released as version 1.4.1.
